### ISSUE TITLE
Bug in VDC evaluation

### ISSUE THREAD

**Issue by Richar-Du** [EvolvingLMMs-Lab/lmms-eval#379]
Thanks for the authors' contribution to the VDC task! When I try to evaluate LLaVA-OV on detailed_test, it gives the following error:
```
File "lmms-eval/lmms_eval/api/task.py", line 689, in __init__ 
    self._config = TaskConfig(**config)                        
TypeError: TaskConfig.__init__() got an unexpected keyword argument 'model_specific_prompt_kwargs'  
| ERROR    | __main__:cli_evaluate:348 - Error during evaluation: TaskCo│
nfig.__init__() got an unexpected keyword argument 'model_specific_prompt_kwargs'. Please set `-│
-verbosity=DEBUG` to get more information. 
```
Following is my evaluation script:
```
MODEL_PATH=$1
TASK=$2
accelerate launch --num_processes=1 \
-m lmms_eval \
--model llava_onevision \
--model_args pretrained=$MODEL_PATH,conv_template=qwen_1_5,model_name=llava_qwen \
--tasks $TASK \
--batch_size 1 \
--log_samples \
--log_samples_suffix llava_onevision \
--output_path ./logs/
```
where MODEL_PATH is the path to LLaVA-OV, and TASK=detailed_test.
Could you please advise if there's anything that needs fixing? @rese1f

**Comment by kcz358** [comment]
Hello, this has been fixed in #382

**Comment by Richar-Du** [comment]
Thanks!
